### PR TITLE
Cleanup SANS release notes

### PR DESCRIPTION
**Pull request by backmari**
### Description of work

This PR does some cleanup for the SANS release notes so they are ready for the release of Mantid v6.9.

### To test:

Read through the SANS release notes to make sure they make sense/no typos.

---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

### CODE REVIEW

File: docs/source/release/v6.9.0/sans.rst [hunk 1]
```diff
@@ -15,7 +15,10 @@ New Features
 - Switching between a custom output file and an automatically-generated one can now be performed using the
   ``Enter Custom Filename`` checkbox on the :ref:`Sum Runs tab <ISIS_SANS_Sum_Runs_Tab-ref>` in the
   :ref:`ISIS SANS GUI <ISIS_Sans_interface_contents>`
```
Comment (robertapplin) [inline docs/source/release/v6.9.0/sans.rst:17]:
```suggestion
  :ref:`ISIS SANS GUI <ISIS_Sans_interface_contents>`.
```

File: docs/source/release/v6.9.0/sans.rst [hunk 2]
```diff
@@ -30,7 +33,7 @@ Bugfixes
   between 1D and 2D.
 - Background Subtracted (``_bgsub``) workspaces are now correctly deleted when ``File`` is selected as the save
   location in the ISIS SANS GUI.
```
Comment (robertapplin) [inline docs/source/release/v6.9.0/sans.rst:35]: Do you think we should link to the ISIS SANS GUI here?

**Review by robertapplin — CHANGES_REQUESTED** [review]
Thanks for the changes, looks really good - only had a few picky suggestions! @rbauststfc @cailafinn @jclarkeSTFC is there anything you want  to highlight with an image in these release notes?

**Comment by cailafinn** [comment]
> Thanks for the changes, looks really good - only had a few picky suggestions! @rbauststfc @cailafinn @jclarkeSTFC is there anything you want to highlight with an image in these release notes?

Improvements to the beam centre finder:
image

CanSAS1D now disabled when 2D reduction is selected:

image

**Review by robertapplin — APPROVED** [review]
Looks good, the links are working and the images are clear 👍

**Pull request merged**